internal/character/clara/skill.go

```diff
@@ -0,0 +1,26 @@
+package clara
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+func (c *char) Skill(target key.TargetID, state info.ActionState) {
+	c.engine.Attack(info.Attack{
+		Source:     c.id,
+		Targets:    c.engine.Enemies(),
+		DamageType: model.DamageType_PHYSICAL,
+		AttackType: model.AttackType_SKILL,
+		BaseDamage: info.DamageMap{
+			model.DamageFormula_BY_ATK: skill[c.info.SkillLevelIndex()],
+		},
+		StanceDamage: 30.0,
+		EnergyGain:   30,
+	})
```

Comment:
missing revenge hits. Full skill logic is as follows:
1. Hit all enemies that have counter mark (hit `AttackType == SKILL`, does 0 stance damage and gens 0 energy)
2. Hit all enemies (the hit you currently have defined)
3. `EndAttack`
4. Remove all marks if not E1